statefun-examples/statefun-ridesharing-example/statefun-ridesharing-example-simulator/pom.xml

```diff
@@ -92,12 +102,6 @@ under the License.
             <version>28.0-jre</version>
         </dependency>
 
-        <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
```

Comment:
can we separate this commit into 1 part that removes lombok, the other changing the parent pom? The commit message no longer covers the actual changes, and tbh it is a bit messy.

Reply:
I agree that this is a bit messy, because removing Lombok also means replacing the classes with a full blown value classes.
I will separate these changes to two separate commits.